doc/classes/ProjectSettings.xml

```diff
@@ -177,6 +177,14 @@
 				Saves the configuration to a custom file. The file extension must be [code].godot[/code] (to save in text-based [ConfigFile] format) or [code].binary[/code] (to save in binary format). You can also save [code]override.cfg[/code] file, which is also text, but can be used in exported projects unlike other formats.
 			</description>
 		</method>
+		<method name="set_as_basic">
+			<return type="void" />
+			<param index="0" name="name" type="String" />
+			<param index="1" name="basic" type="bool" />
+			<description>
+				Defines if the specified property is considered basic or advanced. Basic properties will always be shown in the project settings. Advanced properties will only be shown if the user enables the "Advanced Settings" option.
```

Comment:
```suggestion
				Defines if the specified setting is considered basic or advanced. Basic settings will always be shown in the project settings. Advanced settings will only be shown if the user enables the "Advanced Settings" option.
```

I know the method description just below here says property but I think it should say setting. Maybe you can fix that description too.